jupyterlab_eduhelx_submission/handlers.py

```diff
@@ -20,25 +20,33 @@
 from .process import execute
 from ._version import __version__
 
+class DependencyContainer:
```

Comment:
Why is this called `DependencyContainer` instead of something like `AppContext`? I feel like that's more accurate to what it is. DependencyContainer is so generic it could be used for anything.

Reply:
It actually was originally named AppContext and I changed it for some reason. I have no problem with changing it back.